StepDefinitions/MyProfileLanguagesStep.cs

```diff
@@ -0,0 +1,125 @@
+using NUnit.Framework;
+using ProjectMars.Pages;
+using ProjectMars.Utilities.Hooks;
+using TechTalk.SpecFlow;
+
+namespace ProjectMars.StepDefinitions
+{
+    /// <summary>
+    /// Step definition for language functionality in my profile.
+    /// </summary>
+    [Binding]
+    public class MyProfileLanguagesStep : Hooks
+    {
+        SignInPage signInPageObj = new SignInPage();
+        ProfilePageLanguages profilePageObj = new ProfilePageLanguages();
+
+        [Given(@"User logs into Mars application")]
+        public void GivenUserLogsIntoMarsApplication()
+        {
+            signInPageObj.SignInToApplication(driver);
+        }
+
+        [When(@"User adds '([^']*)' language with '([^']*)' proficiency level into the known languages list")]
+        public void WhenUserAddsLanguageWithProficiencyLevelIntoTheKnownLanguagesList(string language, string proficiencyLevel)
+        {
+            profilePageObj.AddLanguageWithProficiencyLevel(driver, language, proficiencyLevel);
+        }
+
+        [Then(@"The '([^']*)' language with '([^']*)' proficiency level should appear in the known languages list")]
+        public void ThenTheLanguageWithProficiencyLevelShouldAppearInTheKnownLanguagesList(string language, string proficiencyLevel)
+        {
+            if (profilePageObj.VerifyTheAddedLanguage(driver, language, proficiencyLevel))
```

Comment:
i would also assert the message that you get when you add skill or language

Reply:
Yes, I have asserted all the messages.

![Screenshot 2024-05-23 092455](https://github.com/poornimadayalan/ProjectMars/assets/153768077/31a512ab-90e0-4136-9abd-de074ccf82bf)